docs/configuration/elements.md

```diff
@@ -59,7 +59,8 @@ Global line options: `Chart.defaults.global.elements.line`.
 | `borderDashOffset` | `number` | `0.0` | Line dash offset. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineDashOffset).
 | `borderJoinStyle` | `string` | `'miter'` | Line join style. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/lineJoin).
 | `capBezierPoints` | `boolean` | `true` | `true` to keep Bézier control inside the chart, `false` for no restriction.
-| `fill` | <code>boolean&#124;string</code> | `true` | Fill location: `'zero'`, `'top'`, `'bottom'`, `true` (eq. `'zero'`) or `false` (no fill).
+| `cubicInterpolationMode` | `string` | `''` |  Interpolation mode to apply. [See more...](../charts/line.md#cubicinterpolationmode) 
+| `fill` | <code>boolean&#124;string</code> | `true` | Fill location: `'origin`, `'nofill'`, `'start'`, `'end'`, `true` (eq. `'origin'`) or `false` (no fill). [See more...](../charts/area.md#filling-modes) 
```

Comment:
`'nofill'` is not used anywhere in the code.

Reply:
Thanks, does it mean we have origin, start, end, true and false?

Reply:
You could just duplicate the line from the line chart docs:

```
| `fill` | <code>boolean&#124;string</code> | `true` | How to fill the area under the line. See [area charts](../charts/area.md#filling-modes).
```

Reply:
yes, sorry. My fault. I'll resubmit